spec/features/Dishes/show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe "Dish Show Page" do
+  
+  before(:each) do
+   @chef = Chef.create(name: "Alex")
+   @dish = @chef.dishes.create!(name: "test", description: "im a good dish")
+   @dish_1 = @chef.dishes.create!(name: "test1", description: "im a better dish")
+
+   @ingredient = Ingredient.create!(name: "salt", calories: 12)
+   @ingredient_1 = Ingredient.create!(name: "pepper", calories: 20)
+   @dish_ingredient = DishIngredient.create!(dish_id: @dish.id, ingredient_id: @ingredient.id)
+   @dish_ingredient_1 = DishIngredient.create!(dish_id: @dish.id, ingredient_id: @ingredient_1.id)
+  end
+  
+  it "shows a dish details" do
+    visit dish_path(id: @dish.id)
+
+
+    expect(page).to have_content(@dish.name)
+    expect(page).to have_content(@dish.description)
+    expect(page).to have_content(@ingredient.name)
+    expect(page).to have_content(@ingredient_1.name)
+    expect(page).to have_content(32)
```

Comment:
Consider tying this "32" to its label "total_calorie_count" (which you've set up in your view) so that we can be sure that this number is showing up in the right place and is just a random integer from somewhere else on the page. Your expect could instead be `expect(page).to have_content("total_calorie_count: 32")`